src/preset_cli/cli/superset/sync/dbt/metrics.py

```diff
@@ -399,3 +393,32 @@ def get_models_from_sql(
             raise ValueError(f"Unable to find model for SQL source {table}")
 
     return [model_map[ModelKey(table.db, table.name)] for table in sources]
+
+
+def remove_line_break_wrappers(sql: str) -> str:
+    """
+    Remove leading and trailing line breaks from SQL statements.
+    """
+    pattern = r"^\s*\n|\n\s*$"
+    return re.sub(pattern, "", sql, flags=re.MULTILINE)
```

Comment:
`sqlglot` handles this out of the box, but the fallback approach with `regex` doesn't.

Reply:
Can't we do this with just `sql.strip()`?

Reply:
omg you're 100% right 🤦 talk about overcomplicating things lol lemme fix that real quick